GridPathPlannerROS:
  treat_unknown_as_free: true
